cmd/wsl/main.go

```diff
@@ -32,6 +32,7 @@ func main() {
 
 	flag.BoolVar(&config.StrictAppend, "strict-append", true, "Strict rules for append")
 	flag.BoolVar(&config.AllowAssignAndCallCuddle, "allow-assign-and-call", true, "Allow assignments and calls to be cuddled (if using same variable/type)")
+	flag.BoolVar(&config.AllowAssignAndAnythingCuddle, "allow-assign-and-anything", true, "Allow assignments and anything to be cuddled")
```

Comment:
I don't think it's a good idea to add such a major change defaulted to true. Could this be set to false by default?

Reply:
Sorry, it should be false. My mistake.

Reply:
Fixed that.